cinema/views.py

```diff
@@ -34,6 +36,26 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_int(qs):
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
Move it to the separate file or to the global level because you're duplicating it in line 82